docs/source/orderer/ordering_service.md

```diff
@@ -224,14 +216,13 @@ implementation the node will be used in), check out [our documentation on standi
 * **Solo** (deprecated in v2.x)
 
   The Solo implementation of the ordering service is intended for test only and
-  consists only of a single ordering node.  It has been deprecated and may be
-  removed entirely in a future release.  Existing users of Solo should move to
+  consists only of a single ordering node. It has been deprecated and may be
+  removed entirely in a future release. Existing users of Solo should move to
   a single node Raft network for equivalent function.
 
 ## Raft
 
-For information on how to configure a Raft ordering service, check out our
-[documentation on configuring a Raft ordering service](../raft_configuration.html).
+For information on how to customize the `orderer.yaml` file that determines the configuration of an ordering node, check out the [Checklist for a production ordering node](../deployorderer/ordererchecklist.html).
```

Comment:
Not sure if it matters but why is this line the only one longer than 80 characters? Stands out visually while reviewing.

Reply:
The 80 character line limit was a formatting necessity in the gerrit days. If lines went on for longer they stretched out to the right to infinity. 

Now that we're reviewing on github that's line limit is no longer necessary --- github handles the lines sensibly. I decided not to go back and change all of the line lengths because it would hamper the review process, but for lines I substantially rewrote it also made no sense to me to keep the old line length restriction.

Reply:
Gotcha, makes sense.